web/package.json

```diff
@@ -29,8 +29,12 @@
     "postinstall": "npm-license-crawler -onlyDirectDependencies -json ./src/assets/licenses.json"
   },
   "dependencies": {
+    "@emotion/react": "^11.14.0",
+    "@emotion/styled": "^11.14.0",
```

Comment:
Do we need emotion here? or this is a preparation for migrating to emotion too?

Reply:
Mui uses emotion as styling library per default, see also https://mui.com/material-ui/getting-started/installation/#default-installation. Since support for styled components is legacy and the whole library is deprecated, I think it makes lots of sense to migrate to emotion during the style system implementation.